### PR TITLE
REV-398-transaction-flow-step-3

### PR DESCRIPTION
**Pull request by IbrahimAldereni-socuimtech**
### Jira URL

https://optimumpartners.atlassian.net/browse/REV-398

### Description

- finish step 3 upload files   
- backend PR:  https://github.com/abukhadraop-or/reverifi-api/pull/59

### CODE REVIEW

File: src/components/my-profile/MyProfileWrapper/index.jsx [hunk 1]
```diff
@@ -282,7 +282,7 @@ function MyProfileWrapper() {
   const handleUploadImage = async () => {
     // if (!selectedFile) show an error message here;
 
-    await uploadImage({
+    await uploadSingleFile({
```
Comment (mohdabed123) [inline src/components/my-profile/MyProfileWrapper/index.jsx:285]:
```suggestion
    await singleFileUpload({
```

File: src/components/transaction/UploadDocumentsWrapper/index.jsx [hunk 2]
```diff
@@ -78,14 +64,70 @@ function UploadDocumentsWrapper() {
   } = useForm();
 
   /**
-   * on add files function
+   * find documents
+   */
+  const findDoc = (docId) =>
```
Comment (mohdabed123) [inline src/components/transaction/UploadDocumentsWrapper/index.jsx:69]:
```suggestion
  const getDocumentById = (docId) =>
```

File: src/components/transaction/UploadDocumentsWrapper/index.jsx [hunk 3]
```diff
@@ -1,15 +1,13 @@
+/* eslint-disable no-nested-ternary */
```
Comment (mohdabed123) [inline src/components/transaction/UploadDocumentsWrapper/index.jsx:1]:
```suggestion
```

**Review by mohdabed123 — APPROVED** [review]
Small changes

File: src/components/transaction/UploadDocumentsWrapper/index.jsx (hunk 3 above)
Comment (IbrahimAldereni-socuimtech) [reply src/components/transaction/UploadDocumentsWrapper/index.jsx:1]: i need to nest

Comment (mohdabed123) [reply src/components/transaction/UploadDocumentsWrapper/index.jsx:1]: @IbrahimAldereni-socuimtech you can do it without nesting ternary

**Review by mohdabed123 — APPROVED** [review]

**Pull request merged**